src/resources/org/apache/cassandra/cql3/reserved_keywords.txt

```diff
@@ -54,4 +54,5 @@ USE
 USING
 VIEW
 WHERE
-WITH
\ No newline at end of file
+WITH
+CONSTRAINT
```

Comment:
Do we want just `CONSTRAINT` or also include `CHECK`?

Reply:
With the new proposed syntax, nothing else should be required to add here right?